src/Curl/Curl.php

```diff
@@ -146,12 +146,12 @@ class Curl
     /**
      * Constructor ensures the available curl extension is loaded.
      *
-     * @throws \ErrorException
+     * @throws \RuntimeException
```

Comment:
This can break libraries using it. This will involve making a major version. But you are right.

Reply:
A new major version (3.0) must be released with an UPGRADE.md document which contains the changes :-)